tests/unit/type/list.tcl

```diff
@@ -439,6 +973,30 @@ start_server {
       $rd read
     } {}
 
+    test {BRPOPRPUSH timeout} {
```

Comment:
these 3 don't add real value, let's trim them.

p.s. we can improve the original test, which is quite silly (removing the `after` will make no difference).
something like:
```
    $rd brpoplpush foo_list bar_list 1
    after 100
    assert_equal [s blocked_clients] 1
    wait_for_condition 500 100 {
        [s blocked_clients] 1
    } else {
        fail "client wasn't unblocked"
    }
    $rd read
```
if you feel like improving the one, and trimming the 3 dups, that would be great.

Reply:
Removed the dups and improved the existing one as suggested. I used two wait_for_condition calls, I believe it improves it according to what you expected.